newscout_web/frontend/src/app/DashboardApp/ArticleForm.js

```diff
@@ -0,0 +1,274 @@
+import React from 'react';
+import moment from 'moment';
+import ReactDOM from 'react-dom';
+import Select from 'react-select';
+import Datetime from 'react-datetime';
+import { ToastContainer } from 'react-toastify';
+import * as serviceWorker from './serviceWorker';
+import {ARTICLE_SOURCE_LIST_URL, ARTICLE_CATEGORY_LIST_URL,
+    ARTICLE_CREATE_URL, ARTICLE_DETAIL_URL} from '../../utils/Constants';
+import DashboardMenu from '../../components/DashboardMenu';
+import DashboardHeader from '../../components/DashboardHeader';
+import { getRequest, postRequest, putRequest } from '../../utils/Utils';
+import { Button, Form, FormGroup, Label, FormText, Row, Col } from 'reactstrap';
+import Summernote from '../../components/Summernote';
+
+import './index.css';
+
+class ArticleForm extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			modal: false,
+			fields: {
+                title: "",
```

Comment:
Figure out way to automatically form JS code

Reply:
it is already formatted. No need to reformat this.